pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -526,6 +680,107 @@ func getStateData(cs fwk.CycleState) (*stateData, error) {
 	return s, nil
 }
 
+func (pl *DynamicResources) filterExtended(state *stateData, pod *v1.Pod, nodeInfo *framework.NodeInfo, logger klog.Logger) (*resourceapi.ResourceClaim, *fwk.Status) {
```

Comment:
The scheduler framework already has "extensions", this here could be confused with that. Let's be more verbose and use `filterExtendedResources`. Please also add comments to all new methods with the information that reviewers should have to understand what the method is meant to do.

Reply:
sure, renamed, also added comments to describe what the method does.